scripts/add_user_to_group.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import json
+import pprint
+import requests
+
+
+def login_client(client_id, client_secret, url):
+    headers = {'Content-Type': 'application/x-www-form-urlencoded',
+               'Accept': 'application/json'}
+    payload = {'grant_type': 'client_credentials',
+               'response_type': 'token',
+               'token_format': 'opaque'}
+    response = requests.post(f'http://{url}/oauth/token', headers=headers,
+                             params=payload,
+                             auth=(client_id, client_secret))
+    resp_json = response.json()
+    pprint.pprint(resp_json)
```

Comment:
Do we want to print all this?

Reply:
It's useful for debugging but I'll hide it behind a verbose flag